sql/moz-fx-data-shared-prod/mozilla_org_derived/www_site_hits_v2/schema.yaml

```
@@ -0,0 +1,193 @@
+fields:
+- mode: NULLABLE
+  name: date
```

For a more meaningful name, since there a re many dates in the tables.
```suggestion
  name: event_date
```